app/auth.py

```
@@ -0,0 +1,223 @@
+@bp.route('/login')
+def login():
+    """Redirect to Auth0 login page"""
+    # Multiple logging methods to ensure we see this
+    print(">>> LOGIN ROUTE HIT <<<")
+    current_app.logger.error(">>> LOGIN ROUTE HIT <<<")
+    
+    # Write to a simple file as a backup
+    try:
+        with open('/tmp/login_log.txt', 'a') as f:
+            f.write(f"Login hit at {datetime.now()}\n")
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)